src/pages/App.svelte

```diff
@@ -82,6 +83,14 @@
     );
     return geojson;
   }
+
+  async function checkAuthorityValid(authorityName: string): Promise<void> {
+    let authortiesNameSet = await getAuthoritiesNameSet();
```

Comment:
This will cause us to fetch and parse the authorities file twice. Inside `onMount`, we already do `boundaryGeojson = await loadAuthorityBoundary();`. Inside that function, we filter the GJ to look for the authority name. If the result is empty, we have an error -- so we don't even need the name set at all, actually